pages/Filters/index/FilterItemSettings.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="space-y-2 mb-4">
+    <div v-if="filter.keywords.length > 0" class="flex flex-wrap gap-1">
+      <span v-for="(keyword, i) in filter.keywords" :key="i"
+        class="inline-flex items-center px-2 py-1 rounded-md text-xs font-medium bg-slate-100 dark:bg-slate-700 text-slate-700 dark:text-slate-300">
+        {{ keyword }}
+      </span>
+    </div>
+
+    <div v-if="filter.priceRange" class="flex items-center text-sm text-slate-500 dark:text-slate-400">
+      <TagIcon class="w-4 h-4 mr-1" />
+      <span>{{ handleMixMax(filter.priceRange) }}</span>
+    </div>
+    <div v-if="filter.location" class="flex items-center text-sm text-slate-500 dark:text-slate-400">
+      <MapPinIcon class="w-4 h-4 mr-1" />
+      <span>{{ handleLocation(filter.location) }}</span>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { MapPinIcon, TagIcon } from 'lucide-vue-next';
+
+
+defineProps<{ filter: filterFace }>()
+
+function handleMixMax(value: string): string {
```

Comment:
i would call this function something like "get minMaxAsString" or something like that as it doesnt really handle anything. 
A name like this would imply some computation of some kind